pkg/logs/processor/processor.go

```diff
@@ -141,7 +175,18 @@ func (p *Processor) applyRedactingRules(msg *message.Message) bool {
 		}
 	}
 
-	// TODO(remy): this is most likely where we want to plug in SDS
+	// Use the SDS implementation
+	// --------------------------
+
+	// Global SDS scanner, applied on all log sources
+	if p.sds.IsReady() {
```

Comment:
The way I read this, it seems like it is possble for logs to flow through the processor before sds is ready. 
Is that correct (and is that what we want to do)? 

I may be lacking some context here but I would assume if SDS is enabled, we do not want any logs to bypass it